### PR TITLE
Feature/sdl 0264 separating the change of audible status and the change of hmi status

### PR DESCRIPTION
**Pull request by Yuki-Kanadome-Nexty**
Guidelines updates according to the proposal [SDL-0264 Separating the change of Audible status and the change of HMI Status](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md)

This PR is ready for review.

### Summary
Revise PHONE_CALL description and sequence in HMI Integration Guidelines according SDL 0264.

### CODE REVIEW

**Comment by Yuki-Kanadome-Nexty** [comment]
@JackLivio -san
This PR is ready review.
Thank you!

File: docs/BasicCommunication/OnEventChanged/index.md [hunk 1]
```diff
@@ -16,22 +16,22 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 #### PHONE_CALL
 
 !!! MUST
-1. Send notification with appropriate parameter value when active call on HMI has been started/ended.
+1. Change audioStreamingState to NOT_AUDIBLE for all apps (keep hmiLevel) when active call on HMI has been started.
 
-2. Send [SDL.OnAppDeactivated](../../sdl/onappdeactivated) to the active app when the phone call is started.
-
-3. Resume the applications to their original state prior to the phone call event in the HMI when the event ends (see note below).
+2. Send notification with appropriate parameter value when the event ends.
```
Comment (jacobkeeler) [inline docs/BasicCommunication/OnEventChanged/index.md:21]: I would just remove point 2 in the original, no need to rewrite most of this section
```
1. Send notification with appropriate parameter value when active call on HMI has been started/ended.

2. Resume the applications to their original audio streaming state prior to the phone call event in the HMI when the event ends (see note below).
```

File: docs/BasicCommunication/OnEventChanged/index.md [hunk 2]
```diff
@@ -16,22 +16,22 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
...
 !!!
 
 !!! NOTE
-SDL does not send BC.ActivateApp or BC.OnResumeAudioSource to HMI after the phone call is ended.
+- SDL does not send BC.ActivateApp or BC.OnResumeAudioSource to HMI after the phone call is ended.
+
+- If HU wants to switch the screen (HMIStatus) during PHONE_CALL, they can use API `BC.OnAppDeactivated (AppID)` and `BC.OnAppActivated (AppID)`.
```
Comment (jacobkeeler) [inline docs/BasicCommunication/OnEventChanged/index.md:27]:
```suggestion
If the HMI wants to leave the app screen when the PHONE_CALL event occurs, the app's HMI state can be changed accordingly using the `BC.OnAppDeactivated(appID)` and `BC.OnAppActivated(appID)` notifications.
```

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jordynmackool** [comment]
Hi @Yuki-Kanadome-Nexty, please see Jacobs's feedback on the PR and let us know if you have any questions. Thanks!

**Comment by jordynmackool** [comment]
Hi @Yuki-Kanadome-Nexty, to ensure we stay on track for the planned October 27, 2021 release, please respond/update this PR based on the feedback provided by Jacob as soon as possible. Thanks!

File: docs/BasicCommunication/OnEventChanged/index.md [hunk 3]
```diff
@@ -16,22 +16,22 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
...
 !!!
 
 Upon receiving `OnEventChanged(PHONE_CALL)`, SDL will:
 
 |isActive|Result|
 |:-------|:-----|
-|true|Change the HMI state of all media applications currently either in FULL or LIMITED to (BACKGROUND, NOT_AUDIBLE) and all navi/projection apps to (LIMITED, NOT_AUDIBLE, STREAMABLE)|
+|true|Keep the HMI state of all applications but change audible state of all applications to NOT_AUDIBLE|
```
[the remark this answers is not in the document]
Comment (Akihiro-Miyazaki) [reply docs/BasicCommunication/OnEventChanged/index.md:34]: Sorry for late response.
We should confirm.
It is possible for us to modify the sentence according to your suggestion.
However, it is agreed by Shala -san, JackLivio -san, and Kawano -san. 
Is there any problem to change it?

File: docs/BasicCommunication/OnEventChanged/index.md (hunk 1 above)
Comment (Akihiro-Miyazaki) [reply docs/BasicCommunication/OnEventChanged/index.md:21]: We think there is no problem to modify according to your suggestion.

File: docs/BasicCommunication/OnEventChanged/index.md (hunk 2 above)
Comment (Akihiro-Miyazaki) [reply docs/BasicCommunication/OnEventChanged/index.md:27]: We think there is no problem to modify according to your suggestion.

**Comment by Akihiro-Miyazaki** [comment]
@jordynmackool -san,
Thank you for tagging me.
Since @Yuki-Kanadome-Nexty is out of this work, we think it is difficult for him to correspond it.
Therefore, if possible, we would like to close these PR and create new PR.
Is it possible?

**Comment by jordynmackool** [comment]
Hi @Akihiro-Miyazaki, I will close this PR now as you have requested. Please submit your new PR as soon as possible with the noted updates called out by Jacob in the comments above. Once the PR is submitted and ready for re-review, please tag me and Livio will review it. Thanks!

**Pull request closed without merging**